src/test/java/org/fundacionjala/pivotal/cucumber/steps/ui/StorySteps.java

```diff
@@ -63,4 +61,44 @@ public void verifytheStoryIsCreated() {
     public void createsOtherAStoryCalled(final String arg1) {
         story.createStory(arg1);
     }
+
+    /**
+     * This method clicks the expand button for a specific story.
+     *
+     * @param storyKeyName the name of the story.
+     */
+    @When("selects the dropdown button of the story {string}")
+    public void selectsTheDropdownButtonOfTheStory(final String storyKeyName) {
```

Comment:
please rename it to expandstory

Reply:
fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0